app/views/organizations/adopter_fosterer/fostered_pets/index.html.erb

```diff
@@ -4,7 +4,7 @@
     <div data-controller="card">
       <div class="row row-cols-1 row-cols-md-2 row-cols-lg-3 g-4 mb-5">
```

Comment:
Do we really need to use grid for this? I think we can just use flex as it's a simple 1 row card.